src/lib/utils.ts

```
@@ -1,7 +1,34 @@
+import {DateTime} from "luxon"
+
+////////////////////////////////////////////////////////////////////////////////
+
 /** Wrap this around a type to get the evaluated types, not the way to build the types that typescript likes to show you */
 export type Debug<T> = T extends Function ? T : {[K in keyof T]: T[K]} // eslint-disable-line @typescript-eslint/no-unsafe-function-type
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)